package.json

```diff
@@ -132,10 +145,12 @@
 		"test": "nyc mocha \"test/**/*.test.js\"",
 		"posttest": "npm run lint",
 		"lint": "eslint --fix ./src ./test",
-		"prepack": "oclif-dev manifest && oclif-dev readme --multi && npm shrinkwrap && git checkout origin/main -- package-lock.json",
+		"clean": "find src/ -type d -name 'dist' -exec rm -rf {} +",
+		"updatemd": "find docs -type f -name '*.md' -exec sed -i '' 's/\\.ts/\\.js/g' {} +",
+		"prepack": "oclif manifest && oclif readme --multi && npm run updatemd && npm shrinkwrap && git checkout origin/main -- package-lock.json",
 		"postpack": "rm -f oclif.manifest.json && rm -f npm-shrinkwrap.json",
-		"version": "oclif-dev readme --multi && git add README.md && git add docs",
-		"build": "oclif-dev pack:macos && rm -rf tmp/ && oclif-dev pack:win && rm -rf tmp/",
+		"version": "oclif readme --multi && git add README.md && git add docs",
+		"build": "oclif pack:macos && rm -rf tmp/ && oclif pack:win && rm -rf tmp/",
```

Comment:
Were the new commands tested for our release process?

Reply:
I think we will have another ticket for testing the release process, but from this point, we can create the binary file already.